{{cookiecutter.project_slug}}/README.md

```diff
@@ -32,4 +32,40 @@ Download the ZIP file of icons that the site generates for you and paste them in
 
 When we run collectstatic the public folder is copied as is and enables serving of the favicons
 
+## Local development with Docker ##
 
+Docker has been added to this app for local development. Using docker can be a replacement during local development as it will run irrespective of your current configurations 
+
+**NOTE: Change DB_HOST in your .env file to use the docker host**
+
+**NOTE: An initial pipfile lock is required for the backend**
+
+The docker configuration is managed using a docker-compose.yaml, to begin you will need to install docker and set up an account with docker hub
+
+If you are starting the project from scratch make sure to install the python packages with pipenv to generate a pipfile.lock 
+
+Additionally each user must run npm install prior to make run to use the local copy of node_modules
+
+Why is the node modules mounted over?
+1. The user does not have to sh into the container to install new packages
+2. The node modules folder does not overwrite the local version 
+3. Your editor can reference the node_modules for its needs (on your local)
+
+For ease of use you can use the commands from the makefile to run the containers (server,client,database server and client can be ran individually) when you make changes to the original images you will need to rebuild them.
+You can checkout all the commands in the [Makefile](./Makefile)
+
+**Print all available commands**
+- make commands
+
+**Run Server-Client-DB (-d for detached mode)**
+- make run 
+- make run-d 
+
+**Run Client (-d for detached mode)**
+- make run-client
+- make run-client-d
+
+**Run Server-DB (-d for detached mode)**
+***The DB will always run with the server because it is a dependency***
+- make run-server
+- make run-server-d
```

Comment:
Good docs. I think a TL/DR version should be at the top of the README and stated as the preferred way to run locally.
(there is also a section of this README about Favicon's that I think should be moved to the README in the client folder)

Reply:
Some of these steps like the pipfile.lock step could be automatically done. Either when someone runs docker for the first time or during the initial cookiecutter step when the repo has just been created.
It's a slow process though so I could see leaving it to the dev who bootstraps. (perhaps it's own `make` step)

Steps like this though are kind of a "burn after reading" step, that are only needed during initial bootstrap and shouldn't clog up the README or scripts after that.